README.md

```diff
@@ -101,6 +101,39 @@ If your environment requires a proxy, define the proxy properties in the JSON fi
   "proxy": "<proxy_host>:<proxy_port>"
 ```
 
+## Exception handling
+
+All the exceptions raised by the OneView inherits from HPOneViewException.
+
+**HPOneViewException** have the following properties:
+- **msg**  has the message string error, extracted from message attribute from OneView response;
+- **oneview_response** has the entire JSON data (as dict) returned by the OneView; it can be ```None```.
+
+##### Exception handling example:
+
+```python
+try:
+    fc_network = oneview_client.fc_networks.get(id)
+except HPOneViewException as e:
+    print(e.msg)
+    if e.oneview_response:
+    	pprint(e.oneview_response)   
+```
+
+For compatibility purpose, the Exception args property is also filled with the error arguments, for example:
```

Comment:
pelase change purpose to **purposes**.
I guess filled isn`t a word that we use regarting variables. Maybe you could write something like:

For compatibility purposes, the Exception args property is defined with the error arguments. For example: